usaspending_api/etl/tests/integration/test_touch_old_awards_for_es.py

```diff
@@ -0,0 +1,317 @@
+
+    yield aw1, aw2
+
+
+@pytest.fixture
+def award_data4(db):
+    defc = mommy.make("references.DisasterEmergencyFundCode", code="P", group_name="covid_19")
+
+    aw1 = mommy.make("awards.Award", fain="abc123")
+    aw2 = mommy.make("awards.Award", fain="zyx987")
+
+    Award.objects.filter(pk=aw1.id).update(update_date=OLD_DATE)  # convoluted line to sidestep auto_now()
+    Award.objects.filter(pk=aw2.id).update(update_date=OLD_DATE)  # convoluted line to sidestep auto_now()
+
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=9, award=aw1, disaster_emergency_fund=defc)
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=9, award=aw2, disaster_emergency_fund=defc)
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=11, award=aw1, disaster_emergency_fund=defc)
+
+    yield aw1, aw2
+
+
+@pytest.fixture
+def award_mixed_periods(db):
+    defc = mommy.make("references.DisasterEmergencyFundCode", code="O", group_name="covid_19")
+
+    aw1 = mommy.make("awards.Award", piid="abc123")
+    aw2 = mommy.make("awards.Award", piid="zyx987")
+    aw3 = mommy.make("awards.Award", piid="ABC321")
+    aw4 = mommy.make("awards.Award", piid="ZYX789")
+
+    Award.objects.filter(pk=aw1.id).update(update_date=OLD_DATE)  # convoluted line to sidestep auto_now()
+    Award.objects.filter(pk=aw2.id).update(update_date=OLD_DATE)  # convoluted line to sidestep auto_now()
+    Award.objects.filter(pk=aw3.id).update(update_date=OLD_DATE)  # convoluted line to sidestep auto_now()
+    Award.objects.filter(pk=aw4.id).update(update_date=OLD_DATE)  # convoluted line to sidestep auto_now()
+
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=10, award=aw1, disaster_emergency_fund=defc)
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=10, award=aw2, disaster_emergency_fund=defc)
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=11, award=aw1, disaster_emergency_fund=defc)
+
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=20, award=aw3, disaster_emergency_fund=defc)
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=20, award=aw4, disaster_emergency_fund=defc)
+    mommy.make("awards.FinancialAccountsByAwards", submission_id=21, award=aw3, disaster_emergency_fund=defc)
+
+    yield aw1, aw2, aw3, aw4
+
+
+def test_happy_path(submissions, award_data1):
+    """Simple test case: update single award record"""
+    today = datetime.now(timezone.utc)
+
+    Award.objects.filter(pk=award_data1).update(update_date=OLD_DATE)
+    original_datetime = Award.objects.get(id=award_data1)
+
+    call_command(SCRIPT_NAME)
+
+    after = Award.objects.get(id=award_data1)
+
+    assert original_datetime.update_date.strftime("%Y-%m-%d") == OLD_DATE, "Sanity check failed"
+    assert original_datetime.update_date != after.update_date, "Award was not properly updated"
+    assert after.update_date - today < timedelta(minutes=1), "New datetime isn't today"
```

Comment:
Could this throw an error if the test is run between 12:00-12:01am?

Reply:
Unless I am mistaken, both of these values are datetimes. Can you walk me through the problem?

Reply:
I looked at this again, and realized I was thinking about it wrong. We're good to go here